sysutils/duti/Portfile

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        moretension duti 1.5.4 duti-
+
+categories          sysutils
+license             public-domain
+maintainers         {gmail.com:newtonne.github @newtonne} openmaintainer
+platforms           darwin
+supported_archs     x86_64
+
+description         Select default apps for documents and URL schemes on macOS.
+
+long_description    ${description}
```

Comment:
This sounds somewhat interesting from `description`, could `long_description` explain it further?

The github page has:

duti is a command-line utility capable of setting default applications for various document types on macOS, using Apple's Uniform Type Identifiers (UTI). A UTI is a unique string describing the format of a file's content. For instance, a Microsoft Word document has a UTI of com.microsoft.word.doc. Using duti, the user can change which application acts as the default handler for a given UTI.

Reply:
Yup. I've added this in my latest commit 👍